webhooks.md

```diff
@@ -0,0 +1,113 @@
+# Webhooks
+
+Currently supported webhooks for operation on integrations. Action is triggered everytime when hotel updates the integration.
```

Comment:
The first sentence is weird, I don't understand it, I think it is missing some verb.

Reply:
`Currently supported webhooks that are triggered everytime when hotel updates the integration.` would be better ?